test/routing/test_data.cc

```diff
@@ -0,0 +1,100 @@
+#include "./test_data.h"
```

Comment:
I think this style has better readability

https://github.com/motis-project/nigiri/blob/master/test/rt/ripta_test.cc

Reply:
as long as the test case is only used in one file, it's sufficient to have it only there (to prevent having to write a `.h` file and a `.cc` file instead of just having the test inline).